python_modules/dagster-externals/dagster_externals_tests/test_external_execution.py

```diff
@@ -77,14 +81,19 @@ def script_fn():
             ExternalExecutionEnvContextLoader,
             init_dagster_externals,
         )
+        from dagster_externals._io.s3 import ExternalExecutionS3MessageWriter
 
-        context_loader = (
-            ExternalExecutionEnvContextLoader()
-            if os.getenv("CONTEXT_INJECTOR_SPEC") == "user/env"
-            else None
-        )
+        if os.getenv("CONTEXT_INJECTOR_SPEC") == "user/env":
+            context_loader = ExternalExecutionEnvContextLoader()
+        else:
+            context_loader = None  # use default
+
+        if os.getenv("MESSAGE_READER_SPEC") == "user/s3":
+            message_writer = ExternalExecutionS3MessageWriter()
+        else:
+            message_writer = None  # use default
 
-        init_dagster_externals(context_loader=context_loader)
+        init_dagster_externals(context_loader=context_loader, message_writer=message_writer)
         context = ExternalExecutionContext.get()
         context.log("hello world")
```

Comment:
We're not really testing the interval at all here - at least by glancing at what's in our S3 bucket, we only have a single file. It'd be nice to use a short enough interval/long enough log to have confidence in writing/reading multiple objects.

Reply:
That's true, I just tacked onto our standard test intended to test basic functionality of all externals IO. I'll add a test that stresses blob this interval-based blob approach specifically.

Reply:
Actually to keep things simple for now (there is a downstack PR that needs to stabilize) I just modified the existing test to use a short interval and add a `time.sleep` in the external test.